.eslintrc.js

```diff
@@ -50,7 +49,6 @@ module.exports = {
     '@typescript-eslint',
     'react-hooks',
     'react',
-    'prettier',
```

Comment:
ESLint no longer runs the prettier rules, it now purely disables formatting rules that would conflict with prettier.